images/test-runner/rootfs/Dockerfile

```diff
@@ -48,30 +64,23 @@ RUN apk add --no-cache \
   py-pip \
   unzip \
   openssl \
-  cfssl@testing
-
-ENV GOPATH /go
-ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
-
-RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
-
-RUN go install github.com/onsi/ginkgo/v2/ginkgo@v2.9.0  && go install golang.org/x/lint/golint@latest
+  cfssl@testing \
+  curl \
+  tzdata \
+  libc6-compat
 
-ARG RESTY_CLI_VERSION
-ARG RESTY_CLI_SHA
+RUN go install -v github.com/onsi/ginkgo/v2/ginkgo@v${GINKGO_VERSION}
+RUN go install golang.org/x/lint/golint@${GOLINT_VERSION}
```

Comment:
Installing them in the same line can reduce the number of layers.

Reply:
Yea, I can fix that. One of them had an Ossie in testing and I separated them into their own